go/vt/vttablet/tabletserver/throttle/throttler_test.go

```diff
@@ -382,3 +383,51 @@ func TestProbesWhileOperating(t *testing.T) {
 		})
 	})
 }
+
```

Comment:
This here is mostly copy+paste from the now deleted `throttler_exclude_race_test.go` file, and with a few minor changes. Notably `atomic.LoadInt64(...)` and some cosmetics.